README.md

```diff
@@ -80,33 +80,72 @@ metric exporters, service monitors, and custom dashboards.
+
+Use the `USER_DIR` environment variable to specify the root location of your local customization files.
+
+This example sets the `USER_DIR` environment variable to the *my-viya4mon-user-dir* directory:
+
+```bash
+export USER_DIR=~/my-viya4mon-user-dir
+```
+The monitoring and logging deployment scripts use the customization files contained in the directories user the `USER_DIR` location.
+
+### <a name="customization"></a>Specifying Customization Variables and Parameters
+
+After you create the location for your customization files, you can customize the components that you deploy for monitoring and logging by specifying environment variables and Helm chart parameters in this set of customization files (in this example, contained in the *my-viya4mon-user-dir* root directory):
 
 ```text
-- user.env
-- [monitoring]
-  - user.env
-  - user-values-prom-operator.yaml
-  - user-values-pushgateway.yaml
-- [logging]
-  - user.env
-  - user-values-elasticsearch-open.yaml
-  - user-values-es-exporter.yaml
-  - user-values-fluent-bit-open.yaml
+/my-viya4mon-user-dir/user.env
+
+/my-viya4mon-user-dir/monitoring/user.env
+/my-viya4mon-user-dir/monitoring/user-values-prom-operator.yaml
+/my-viya4mon-user-dir/monitoring/user-values-pushgateway.yaml
+
+/my-viya4mon-user-dir/logging/user.env
+/my-viya4mon-user-dir/logging/user-values-elasticsearch-open.yaml
+/my-viya4mon-user-dir/logging/user-values-es-exporter.yaml
+/my-viya4mon-user-dir/logging/user-values-fluent-bit-open.yaml
 ```
 
+You specify the environment variables in the `user.env` files and the Helm chart parameters in the `*.yaml` configuration files. 
+
+You do not have to specify values in all of these files. The deployment process ignores any files in the `USER_DIR` other than these.
+
+#### Specifying Environment Variables in user.env Files
+
+Environment variables control script behavior and high-level options such as TLS and workload node placement. Note that you can also specify environment variables on a command line, but specifying the variables in `user.env` is recommended, in order to maintain a consistent set of values for future deployments. The values in the top-level `user.env` file (`/my-viya4mon-user-dir/user.env`) apply to both the monitoring and logging deployments. The values in `/my-viya4mon-user-dir/monitoring/user.env` apply only to the monitoring deployment and the values in `/my-viya4mon-user-dir/logging/user.env` apply only to the logging deployment.
+
+#### Using Sample Configuration Files
+
+This repository contains several sample configuration files which provide values for specific situations. The `generic-base` sample contains a complete set of customization files and comments in each of the files to enable you to understand the how the values are used. A best practice is to use the files in the `generic-base` sample as a base for your customization files, and then change the values or copy values from other samples to match your situation.  
```

Comment:
Link to generic-base sample (if links to folders like that work)

Reply:
I don't think I can link to a folder, but I liked to the generic-base README, which I think is close enough.